mlflow/utils/_capture_modules.py

```diff
@@ -83,16 +83,21 @@ def main():
     # Mirror `sys.path` of the parent process
     sys.path = json.loads(args.sys_path)
 
+    spark = None
     if flavor == mlflow.spark.FLAVOR_NAME and is_in_databricks_runtime():
```

Comment:
shall we remove the `is_in_databricks_runtime()` condition here ?

Reply:
why?

Reply:
The code in this if caluse only makes sense in Databricks runtime, doesn't it?

Reply:
Make sense.
I think we can remove lines of:
```
        spark = (
            SparkSession.builder.master("local")
            .appName(__name__)
            .config("spark.ui.enabled", "false")
            .getOrCreate()
        )
```
later in `mlflow.spark._load_pyfunc` it will execute similar create session code.

Reply:
Makes sense. Removed the lines to create a spark session!